packages/web-components/src/components/va-alert/va-alert.tsx

```diff
@@ -182,15 +181,16 @@ export class VaAlert {
     if (uswds) {
       const classes = classnames('usa-alert', `usa-alert--${status}`, {
         'bg-only': backgroundOnly,
-        'usa-alert--success': (status === 'continue'),
-        'usa-alert--slim': slim
+        'usa-alert--success': status === 'continue',
+        'usa-alert--slim': slim,
       });
       return (
         <Host>
           <div
             role={this.el.getAttribute('data-role') || role}
             aria-live={ariaLive}
             class={classes}
+            aria-label={this.el.getAttribute('data-label') || role || null}
```

Comment:
@eselkin Just a very minor thing on this line:

`
aria-label={this.el.getAttribute('data-label') || role || null}
`

The `role` value is getting assigned like this:
```
const role = status === 'error' ? 'alert' : null;
```

So I don't think there's any need to add that extra `|| null` in the aria-label assignment? This is for both places.